doc/userguide/rules/datasets.rst

```diff
@@ -34,8 +31,8 @@ Example of sets for tracking unique values::
 
 Rules to go with the above::
 
-    http.user_agent; dataset:set,ua-seen;
-    dns.query; to_sha256; dataset:set,dns-sha256-seen;
+    alert http any any -> any any (msg: "http user-agent test"; http.user_agent; dataset:set,ua-seen; sid:234; rev:1;)
+    alert dns any any -> any any (msg:"dns list test"; dns.query; to_sha256; dataset:isset,dns-sha256-seen; sid:123; rev:1;)
```

Comment:
Better to use
```
.. container:: example-rule
```

Reply:
See rules/header-keywords.rst for some examples of how it is used.

Reply:
updated thanks!